docs/topics/multiplatform-mobile/multiplatform-mobile-understand-project-structure.md

```diff
@@ -253,11 +253,10 @@ The configuration of Android library is stored in the `android {}` top-level blo
 
 ```kotlin
 android {
-    compileSdk = 29
-    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
+    namespace = "com.example.kotlinmultiplatformsandbox"
```

Comment:
Maybe we could omit all the specific configuration inside, and just say something like:

```
android {
    // ... typical Android configuration, refer to Android docs for details ...
}
```

The reason why I'm suggesting this is because:
a) the current configuration provides nothing really useful for users. It's not the minimal configuration, it's not the configuration necessary for KMP specifically, it's just "some configuration"

b) because we're not controlling the DSL in this block, there's a high chance that it will be getting outdated from time to time

Reply:
Right, it's just an example. I suggest mentioning Android docs, but leaving these properties as an example, because we constantly get feedback that we don't have enough examples in our docs

Reply:
I personally prefer when the sample contains everything required, so I copy-paste and click import. So, voting to keep the settings for `android {}`.

Reply:
Hey there,
I think leaving just an empty block makes this code block useless (it can be easily replaced with a simple inline code).
What do we want to show users here?
* If we want just to say about the block, let's inline the code and add a note about Android's documentation.
* If we want to share an example of configuration, just to show what it can look like, let's add an example, like Danil suggested.

Both options are valid.

Reply:
> What do we want to show users here?

Is it a question to me? :) I don't know, I'm not the original author nor the updater. If you want my opinion, I just think that up-to-date example is better than inline note about `android`-block, but outdated example is significantly worse than inline note. That's why I've asked what's our process on handling the up-to-date-ness